tests/context_mt5_test.py

```diff
@@ -52,13 +46,10 @@ def test_connect_login_errors(self, mock_init, mock_login):
         mock_login.return_value = False
         with pytest.raises(PermissionError):
             self.context_with_complete_creds.connect()
-
-    def test_connect_setting_value_missing(self):
-        context_with_no_settings = ContextMT5({}, self.credentials)
-        with pytest.raises(KeyError):
-            context_with_no_settings.connect()
     
+    # I think this is what you mean
```

Comment:
Yes. Please remove this comment before pushing :D
You can always leave comments like this on the PR so that you don't have to do another commit to remove them.